docs/feature-flags.md

```diff
@@ -111,6 +111,10 @@ This lists the feature flags and their sub-configurations to enable/disable and
     - `observability-red-hat-sso-metrics-secret-file`[Required]: The path to the file containing the client
     secret for the metrics service account for use with Red Hat SSO.
 
+### Observability CloudWatch Logging
+  - `observability-enable-cloudwatchlogging`:  Enable Observability to deliver logs to AWS CloudWatch
```

Comment:
Re:
```
Enable Observability to deliver logs to AWS CloudWatch
```

I am wondering what do you think of the below alternative suggestion 
```
Enables the configuration CloudWatch Logging secrets so that Observability can deliver data plane logs to AWS CloudWatch.
```

Reply:
I thought about that but from the point of view of KFM it enables/disables Observability to do that. Other actions other than secrets might be included, even if now it is limited to secrets, so I kept the description independent of that

Reply:
Okay thanks, I am fine with taken the generic approach. 
I'll just propose that we clarify that the logs in question here are data plane logs.
```
Enable Observability to deliver data plane logs to AWS CloudWatch
```

Reply:
:+1: . Observability is only installed on the data plane side so it is assumed that it happens in the dataplane, but it does not harm to further clarify.
I will update the documentation

Reply:
Updated

Reply:
Thanks.